### ISSUE TITLE
Can you share config.json?

### ISSUE THREAD

**Issue by yefx** [Mohamed-sm/Freqtrade-RLStrategy-IA#1]
Can you share config.json?

**Comment by rogue2021** [comment]
much appreciated if you can share your config.json file so we can follow and setup the bot. Thank you

**Comment by DarkReaper112** [comment]
{
    "$schema": "https://schema.freqtrade.io/schema.json",
    "position_adjustment_enable": true,  
    "trading_mode": "futures",
    "margin_mode": "isolated",
    "max_open_trades": 6,
            "timeframe": "15m",
    "stake_currency": "USDT",
    "stake_amount": "unlimited",
    "tradable_balance_ratio": 1,
    "fiat_display_currency": "USD",
    "dry_run": true,
    "dry_run_wallet": 2000,
    "cancel_open_orders_on_exit": false,
    "unfilledtimeout": {
        "entry": 10,
        "exit": 10
    },
    "order_types": {
        "entry": "limit",
        "exit": "market",
        "emergency_exit": "market",
        "force_exit": "market",
        "force_entry": "market",
        "stoploss": "market",
        "stoploss_on_exchange": false,
        "stoploss_on_exchange_interval": 120
    },
    "entry_pricing": {
        "price_side": "same",
        "use_order_book": true,
        "order_book_top": 1,
        "price_last_balance": 0.0,
        "check_depth_of_market": {
            "enabled": false,
            "bids_to_ask_delta": 1
        }
    },
    "exit_pricing": {
        "price_side": "other",
        "use_order_book": true,
        "order_book_top": 1
    },
    "exchange": {
        "name": "binance",
        "key": "",
        "secret": "",
        "ccxt_config": {},
        "ccxt_async_config": {
        },
        "pair_whitelist": [
            "BTC/USDT:USDT",
            "ETH/USDT:USDT"

        ],
        "pair_blacklist": []
    },

        "pairlists": [
            {
                "method": "StaticPairList"
            }
        ],
       "freqai": {
        "enabled": true,
        "conv_width": 1,
        "purge_old_models": 1,
        "expiration_hours": 2,
        "train_period_days": 12,
        "backtest_period_days":4,
        "live_retrain_hours": 1,
        "write_metrics_to_disk": true,
        "identifier": "IQN",
        "fit_live_predictions_candles": 500,
        "data_kitchen_thread_count": 32,
        "track_performance": true,
        "save_backtest_models": true,
        "data_split_parameters": {
            "test_size": 0.3,
            "random_state": 42
        },
        "feature_parameters": {
            "include_corr_pairlist": [
                "BTC/USDT:USDT",
                "ETH/USDT:USDT"
            ],
            "include_timeframes": [
                "15m",
                "30m",
                "1h",
                "4h"
            ],
            "indicator_periods_candles": [
                10,
                20
            ],
            "label_period_candles": 20
        },
        "model_training_parameters": {
            "n_envs": 32,
            "learning_rate": 3.0e-4,
            "lr_anneal" : true,
            "adam_epsilon" : 1e-5,
            "epsilon_decay": 0.995,
            "epsilon_start": 0.4,
            "epsilon_end": 0.01,
            "vf_scale_epsilon" : null,
            "clip_grad_norm": 40.0,
            "buffer_size": 1000000,
            "batch_size": 32,
            "num_quantiles": 32,
            "embedding_dim": 32,
            "hidden_dim": 128,
            "seq_length": 20,
            "num_heads": 8,
            "nstep_train" : 20,
            "nstep_target" : 20,
            "rnn_bootstrap" : true,
            "optimizer_step_size": 500,
            "optimizer_gamma": 0.9,
            "n_jobs": 32,
            "device": "cuda",
            "gamma": 0.99,
            "seed": 42

        },
        "rl_config": {
            "log_every": 500,
            "hist_every": 2000,
            "updates_per_step": 4,    
            "tensorboard_plot": true,
            "plot_new_best": true,
            "train_cycles": 20,
            "max_trade_duration_candles": 300,
            "model_type": "IQN",
            "policy_type": "MlpLstmPolicy",
            "model_reward_parameters": {
                "clip_rewards": true,
                "win_reward_factor": 2,
                "invalid_action_penalty": -2,
                "inaction_penalty": -1
            }
        },
        
        
    },
    "api_server": {
        "enabled": true,
        "listen_ip_address": "127.0.0.1",
        "listen_port": 8080,
        "verbosity": "error",
        "enable_openapi": false,
        "jwt_secret_key": "8f6cd3511259293ff1db72f562fe6e8cc2dfe08c7678843f36f6719ba6b8f2ba",
        "ws_token": "te78Pk3YKO7KKUUWGyHAK1m8LmwZv1U-CQ",
        "CORS_origins": [],
        "username": "freqtrader",
        "password": "1234"
    },
    "bot_name": "freqtrade",
    "initial_state": "running",
    "force_entry_enable": false,
        "internals": {
            "process_throttle_secs": 5
        }
    }
